qiskit/quantum_info/states/quantum_state.py

```diff
@@ -89,6 +89,11 @@ def is_valid(self, atol=None, rtol=None):
         """Return True if a valid quantum state."""
         pass
 
+    @abstractmethod
+    def is_entangled(self, epsilon: float = 1e-10):
```

Comment:
it is better to use the existing atol/rtol parameters than epsilon (see the `is_valid` function above)